src/parser/jobs/mnk/modules/RiddleOfFire.js

```diff
@@ -15,13 +16,18 @@ const ROF_DURATION = STATUSES.RIDDLE_OF_FIRE.duration * 1000
 
 const POSSIBLE_GCDS = 9
 
+const ROF_GCD_TARGET = 9
```

Comment:
Kinda makes sense to make this into an object so it's cleaner and could potentially be used as tiers in the future.